src/components/App/App.js

```diff
@@ -33,22 +41,32 @@ class App extends Component {
           <NavBar />
         </header>
         <Switch>
-          <Route path='/birds/:id' render={({ match }) => {
-            const bird = this.state.regionBirds.find(bird => bird.speciesCode === match.params.id)
-
-            if(!bird) {
-              return (<h2>That bird doesn't exist!</h2>)
-            }
-            return <BirdDetails bird = {bird} />
-          }}>
-            
-          </Route>
-          <Route path='/birds'>
+          <Route
+            path="/birds/:id"
+            render={({ match }) => {
+              const bird = this.state.regionBirds.find(
+                (bird) => bird.speciesCode === match.params.id
+              );
+
+              if (!bird) {
+                return <h2>That bird doesn't exist!</h2>;
+              }
+
+              return <BirdDetails bird={bird} addBird={this.addBird} />;
+            }}
+          ></Route>
+          <Route path="/birds">
             {loadingMsg}
-            <BirdList birdData={this.state.regionBirds}/>
+            <BirdList birdData={this.state.regionBirds} />
+          </Route>
+          <Route path="/myBirds">
+            <MyBirds birdData={this.state.myBirds}/>
```

Comment:
This might be a good place to refactor in the future - using BirdList instead of MyBirds.

Reply:
I love this idea! The two are very similar, this would be a great place to DRY up the code.

Reply:
ok